geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1929,7 +1929,8 @@
    * The static String definition of the <i>"redis-bind-address"</i> property <a
    * name="redis-bind-address"/a>
    * </p>
-   * <U>Description</U>: Specifies the bind address used by Redis API for Geode
+   * <U>Description</U>: Specifies the bind address used by Redis API for Geode. If "" or no bind
```

Comment:
For consistency with most network services, why not treat empty/unspecified bind address to bind to all available addresses?

Reply:
I made a PR for it: https://issues.apache.org/jira/browse/GEODE-8158